### PR TITLE
Update URI macros for the Network Table Listener

### PR DESCRIPTION
**Pull request by jahn18**

### CODE REVIEW

File: projects/network_table_listener/NetworkTableListener.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    """
+
+    def getWindSensorData(self, node, uri):
+        wind_sensor_data = json.dumps(
+            {
+                'sensor_type': 'wind',
+                'sensor_id': uri,
+                'speed': node.children['iimwv'].children['wind_speed'].value.int_data,
+                'direction': node.children['iimwv'].children['wind_direction'].value.int_data
+                #'current': node.children['uccm'].children['current'].value.int_data,
```
Comment (briellelaw) [inline projects/network_table_listener/NetworkTableListener.py:35]: I know it's dumb, but can you just add a space after all the '#' - that's what failing the static_analysis

File: projects/network_table_listener/NetworkTableListener.py [hunk 2]
```diff
@@ -0,0 +1,239 @@
+from nt_connection.Connection import ConnectionError
```
Comment (briellelaw) [inline projects/network_table_listener/NetworkTableListener.py:1]: Can you also add a brief summary about what this file does at the top. 
Example from land_satellite_listener:

```
"""land_satellite_listener.py: Facillitates communication between Land Server and BBB.""" 
   
 __author__ = "Brielle Law (briellelaw), John Ahn (jahn18)" 
 __copyright__ = "Copyright 2020 UBC Sailbot"  
```

File: src/python/uri.py [hunk 3]
```diff
@@ -34,58 +43,71 @@
...
 WINCH = "/actuation_angle/winch"
+WINCH_MAIN = "/actuation_angle/winch/winch_main"
+WINCH_JIB = "/actuation_angle/winch/winch_jib"
 WINCH_MAIN_ANGLE = "/actuation_angle/winch/winch_main/angle"
 WINCH_JIB_ANGLE = "/actuation_angle/winch/winch_jib/angle"
 
 # UCCM
 # TODO: need to update these eventually
-# boom sensor
+# boom sensora
```
Comment (briellelaw) [inline src/python/uri.py]: can you remove the 'a' pls :)) thanks!

**Review by briellelaw — COMMENTED** [review]
After these minor changes, can you also squash your commits

File: projects/network_table_listener/NetworkTableListener.py (hunk 1 above)
Comment (jahn18) [reply projects/network_table_listener/NetworkTableListener.py:35]: done

File: projects/network_table_listener/NetworkTableListener.py (hunk 2 above)
Comment (jahn18) [reply projects/network_table_listener/NetworkTableListener.py:1]: done

File: projects/network_table_listener/NetworkTableListener.py (hunk 1 above)
Comment (jahn18) [reply projects/network_table_listener/NetworkTableListener.py]: done

**Pull request merged**